src/components/PostCard/PostHeader/PostHeader.js

```diff
@@ -100,6 +101,10 @@ export default class PostHeader extends PureComponent {
       timeWindow = `${type} starts ${startDate}`
     }
 
+    let locationZero = (locationText ? locationText.split(',')[0] : '')
```

Comment:
this is to display the post location as city, state when posts are displayed in the stream

Reply:
ah, so lets think through this a bit more because location could be entered in various formats and this might not work. for example if they enter a zoom link then this would show the zoom link and then a comma. one thing we can do is check whether the post has a location object, then we know it was a successfully geocoded location, then you could display {location.city}, {location.region} to get city, state. If the location object does not exist (no geocoded location) then maybe just display the whole locationText? is the concern that it will be too long? or that we dont want to show the exact location here?